Tests/OptimizelyTests-Common/DecisionServiceTests_Features.swift

```diff
@@ -127,32 +129,78 @@ class DecisionServiceTests_Features: XCTestCase {
     var sampleRolloutData: [String: Any] { return
         [
             "id": kRolloutId,
-            "experiments": [sampleRolloutExperimentData]
+            "experiments": sampleRolloutExperimentData
         ]
     }
     
-    var sampleRolloutExperimentData: [String: Any] { return
+    var sampleRolloutExperimentData: [[String: Any]] { return
         [
-            "status": "Running",
-            "id": kRolloutExperimentId,
-            "key": "rolloutExp",
-            "layerId": "10420273888",
-            "trafficAllocation": [
-                [
-                    "entityId": "10389700000",
-                    "endOfRange": 10000
-                ]
+            [
+                "status": "Running",
+                "id": kRolloutExperimentId,
+                "key": "rolloutExp",
+                "layerId": "10420273888",
+                "trafficAllocation": [
+                    [
+                        "entityId": "10389700000",
+                        "endOfRange": 10000
+                    ]
+                ],
+                "audienceIds": [kAudienceIdAge],
+                "variations": [
+                    [
+                        "variables": [],
+                        "id": "10389700000",
+                        "key": kVariationKeyA,
```

Comment:
Let's use different key names (like "kRolloutVariationKeyA"). It's already used in experiment variations. Can be confused. Same for other rollout variations as well.